frontend/PreferencesPanel.js

```diff
@@ -90,7 +90,7 @@ class PreferencesPanel extends React.Component {
               ref={this._setSelectRef}
               value={themeName}
             >
-              {!showHiddenThemes && (<option value="">{browserName}</option>)}
+              {browserName && (<option value="">{browserName}</option>)}
```

Comment:
> It may just be a unilateral request, not sure if people need to go back to default theme for `react-devtools` package. (It had a `Reset` button but removed)

Yeah, I thought replacing the "Reset" button with the name of the browser, particularly given the addition of the new "Custom" option, made more sense in terms of UX. (Subjective call.)